pkg/daemon/update.go

```diff
@@ -1169,23 +1169,25 @@ func (dn *Daemon) deleteStaleData(oldIgnConfig, newIgnConfig *ign3types.Config)
 	return nil
 }
 
-// enableUnit enables a systemd unit via systemctl
-func (dn *Daemon) enableUnit(unit ign3types.Unit) error {
-	_, err := exec.Command("systemctl", "enable", unit.Name).CombinedOutput()
+// enableUnits enables a set of systemd units via systemctl, if any fail all fails.
+func (dn *Daemon) enableUnits(units []string) error {
```

Comment:
Either here or in the caller, we should skip the `exec.Command` if the list is empty.